govcd/task.go

```diff
@@ -79,14 +79,23 @@ func (task *Task) WaitInspectTaskCompletion(inspectionFunc InspectionFunc, delay
 			return fmt.Errorf("error retrieving task: %s", err)
 		}
 
+		// If an inspection function is provided, we pass information about the task processing:
+		// * The task itself
```

Comment:
Nit: convert to lower case "The" -> "the"

Reply:
fixed